docs/cli.md

```diff
@@ -0,0 +1,447 @@
+                  
+
+# BigFlow CLI
+
+BigFlow offers a command-line tool called `bf`.
+It lets you run, build, and deploy your workflows from command-line on any machine with Python.
+
+`bf` is a recommended way of working with BigFlow
```

Comment:
BigFlow CLI is a recommended way of working with BigFlow projects on a local machine...